supervisor-services-labs/argocd-operator/argocd-instance.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: my-demo-argocd
+  namespace: ${{MY_VSPHERE_NAMESPACE}}
```

Comment:
I would remove the namespace and use the kubectl commands to explain how to login + set context + apply?

Reply:
Will do so. Makes sense.